pyproject.toml

```diff
@@ -96,12 +96,12 @@ version = "0.3.0"
 
 [tool.poetry.dependencies]
 PyYAML = "^6.0"
+aiobotocore = { version = "^2.4.0", extras = ["boto3"] }
```

Comment:
@bushwhackr `aiobotocore` is not added here so we don't need to pin version any more